catalog/opensearch/catalog-opensearch-endpoint/src/test/java/org/codice/ddf/endpoints/OpenSearchEndpointTest.java

```diff
@@ -114,19 +110,15 @@ public void testProcessQueryForProperHandlingOfSiteNameLOCAL()
     // Check on the sites passed in to framework.query
     when(mockFramework.query(any(QueryRequest.class)))
         .thenAnswer(
-            new Answer<Object>() {
-              public Object answer(InvocationOnMock invocation) {
-                QueryRequest queryRequest = (QueryRequest) invocation.getArguments()[0];
-
-                // ***Test verification***
-                // This assert is the whole point of this unit test
-                Assert.assertTrue(
-                    ((OpenSearchQuery) queryRequest.getQuery())
-                        .getSiteIds()
-                        .contains(testSiteName));
-
-                return new QueryResponseImpl(queryRequest);
-              }
+            invocation -> {
+              QueryRequest queryRequest = (QueryRequest) invocation.getArguments()[0];
+
+              // ***Test verification***
+              // This assert is the whole point of this unit test
+              Assert.assertTrue(
```

Comment:
ideally we use `assertThat` here.  not sure the comment is useful above

Reply:
There are many of these `Assert` methods in the unit tests in these opensearch bundles that I didn't touch, and it's out of the scope of this ticket to fix all of those.